internal/backends/mods36/encoder.go

```diff
@@ -162,9 +162,9 @@ func (e *Encoder) EncodePublication(p *models.Publication) ([]byte, error) {
 	for _, c := range p.Author {
 		addContributor(r, "editor", c)
 	}
-	// TODO use promoter terminology everywhere?
+
 	for _, c := range p.Supervisor {
-		addContributor(r, "promoter", c)
+		addContributor(r, "Supervisor", c)
```

Comment:
This exporter is using the marcrelator vocabulary which uses promoter as a term, be careful when replacing terminology that is not user facing.

Reply:
Thanks, I'll be more paranoid about it!